.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 jdk: openjdk8
 import:
 - source: ".travis/release-flow.yml"
-  if: branch = master and type = push
+  if: branch = master and type = push and commit_message !~ /(\[skip release\])/
```

Comment:
Please add description of the flag in CONTRIBUTING.md

Reply:
Added